onehungrymind/sermon-beacon
# SB-8: Seed Data Migrations

## Feature/Problem
Seed data for development


## Solution
Add seed data via migrations


## Areas of Impact
API


## UI Images
N/A


## Manual Testing
- Pull down the branch, run the server, open the Hasura console and make sure the data exists.
- Verify CI passes
